backend/lib/github/index.js

```diff
@@ -16,125 +16,59 @@
 
 'use strict'
 
-const _ = require('lodash')
-const axios = require('axios')
-const https = require('https')
-const logger = require('../logger')
+const octokit = require('./octokit')()
+
 const config = require('../config')
-const parseLink = require('parse-link-header')
-const { URL } = require('url')
-const urljoin = require('url-join')
+const {
+  org: owner = 'kubernetes-dev',
+  repository: repo = 'journal-dev'
```

Comment:
do we have to default the org and repo? As I see we also have default values in config/gardener.js